pkg/chartmuseum/routes.go

```diff
@@ -41,3 +41,10 @@ func (server *Server) setRoutes(username string, password string, enableAPI bool
 		writeAccessGroup.DELETE("/api/charts/:name/:version", server.deleteChartVersionRequestHandler)
 	}
 }
+
+func (server *Server) setMultiTenancyRoutes() {
+	server.Router.GET("/", func(c *gin.Context) {
+		msg := "This ChartMuseum install is running in multi-repo mode."
```

Comment:
Are we going with `multi-repo` or `multi-tenant`? Should probably be consistent everywhere either way.

Reply:
Should also put a message saying that its pre-alpha and not currently expected to work.

Reply:
Also is it worth having a separate package for the multi-tenancy code? As long as you have a standard interface it will keep it more modular and more readable.